src/components/HomePage/Banner.js

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import { useTheme, makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import Box from '@material-ui/core/Box';
+import PropTypes from 'prop-types';
+import Countdown from 'react-countdown';
+
+import SvgImg from '../SvgImg';
+import hothLogo from './hoth7-logo.svg';
+import hothBanner from './hoth-banner.svg';
+
+const hothStart = new Date('2021-01-23T09:00:00-07:00');
+
+const useStyles = makeStyles(theme => ({
+	container: {
+		backgroundColor: theme.palette.primary.dark,
+		height: 'auto',
+		[theme.breakpoints.up('md')]: {
+			backgroundImage: `url(${hothBanner})`,
+			backgroundSize: '574px 346px',
+			backgroundRepeat: 'no-repeat',
+			backgroundPosition: 'right bottom'
+		},
+		padding: 25
+	},
+	text: {
+		color: 'white',
+		fontWeight: 500
+	},
+	logo: {
+		width: 150,
+		margin: 20,
+		[theme.breakpoints.down('sm')]: {
+			width: 150
+		}
+	},
+	colon: {
+		color: 'white',
+		fontWeight: 600,
+		fontSize: '1.9em'
+	},
+	timeDesc: {
+		color: 'white',
+		fontWeight: 600,
+		fontSize: '1.9em',
+		'&::after': {
+			color: 'white',
+			content: 'attr(dataField)',
+			fontWeight: 400,
+			fontSize: '0.23em'
+		}
+	}
+}));
+
+function renderInfo(classes) {
+	return (
+		<Grid
+			container
+			direction='column'
+			justify='center'
+			alignItems='center'
+		>
+			<Typography variant='h3' className={classes.text}>Hack on the Hill 7</Typography>
```

Comment:
Can we move the `className` to `Grid`? This way, hopefully we won't have to specify it on every `Typography` component.

Reply:
If we move the `className` to `Grid`, the font gets overridden in the `Typography` component.